gql/transport/requests.py

```diff
@@ -145,7 +147,8 @@ def execute(  # type: ignore
 
         # Pass kwargs to requests post method
         post_args.update(self.kwargs)
-
+        if extra_args:
+            post_args.update(extra_args)
```

Comment:
Please add two newlines, one before the if block and one after

Reply:
> Please add two newlines, one before the if block and one after
fixed